ImageViewer/Source/Extensions/UIButton.swift

```diff
@@ -106,4 +106,12 @@ extension UIButton {
 
         return button
     }
+
+    static func deleteButton() -> UIButton {
+
+        let button = UIButton(frame: CGRect(origin: CGPoint.zero, size: CGSize(width: 80, height: 50)))
+        button.setTitle("Delete", for: UIControlState())
```

Comment:
Yep.  This was a copy/paste from one of the other existing button methods, and I hadn't noticed the `UIControlState()` argument.